### ISSUE TITLE
Bluetooth Barometer?

### ISSUE THREAD

**Issue by tdurack** [Sublimis/UrbanBiker#109]
Does ub support bluetooth Barometer? Would there be any advantage for altitude accuracy and power calc etc?

**Comment by Sublimis** [comment]
Yes, bluetooth barometers, or more formally called "environment sensors", are supported starting from the v5.110 beta! Inbuilt pressure sensors were supported for the last several years.

Sadly however, bluetooth environment sensors are not very common. Can we ask which environment/barometer sensor do you plan to use? If you haven't yet decided, or were unable to find any, we suggest the [RuuviTag](https://ruuvi.com/) environment sensor beacon, as it's the only one tested so far, and proven working.

Using barometer is essential for both altitude and power accuracy (when there's no dedicated power meter). Accuracy improvements over GPS are immense!

**Comment by tdurack** [comment]
Thanks for the response - will look at ruuvitag

**Comment by tdurack** [comment]
Now have a RuuviTag whis seems to work well!

Discovering the sensor is a bit hit-or-miss: typically have to remove from profile and re-discover/add. Not sure if that is android/bt behavior or ub?

The aggravation with remove/add from profile is this disrupts meter selection. I think all meters should be available all the time, regardless of sensor discovery.

Perhaps also mark sensors as bt, so they can be differentiated from built-in sensor? For example: temperature (built-in) vs. temperature (bt)

**Comment by radiotbo** [comment]
> Now have a RuuviTag which seems to work well!
> 
> Discovering the sensor is a bit hit-or-miss: typically have to remove from profile and re-discover/add. Not sure if that is android/bt behavior or ub?
> 
> The aggravation with remove/add from profile is this disrupts meter selection. I think all meters should be available all the time, regardless of sensor discovery.
> 
> Perhaps also mark sensors as bt, so they can be differentiated from built-in sensor? For example: temperature (built-in) vs. temperature (bt)

I think it is a bug from the last version. I also have to remove/add my cadence sensor to be conected.
